src/build.js

```diff
@@ -33,9 +33,11 @@ function bundle(src, dest, argv) {
       // .transform('babelify', {
       //   presets: ['@babel/preset-env'],
       // })
-      .bundle((err, bundle) => {
+      .bundle((err, bundles) => {
```

Comment:
Lets rename the `bundles` variable to `bundleBuffer`. It's a single Node `Buffer` object representing the contents of a single bundle file.